idd/Energy+.idd.in

```diff
@@ -71461,10 +71466,19 @@ SetpointManager:MixedAir,
   A5 , \field Fan Outlet Node Name
        \required-field
        \type node
-  A6 ; \field Setpoint Node or NodeList Name
+  A6 , \field Setpoint Node or NodeList Name
        \required-field
        \type node
        \note Node(s) at which the temperature will be set
+  A7 , \field Cooling Coil Inlet Node Name
+       \type node
```

Comment:
\note Optional field used to limit economizer operation to prevent freezing of DX cooling coil.
*Add same note to next two fields.*